treebrowser/src/treebrowser.c

```diff
@@ -138,11 +138,7 @@ PLUGIN_SET_TRANSLATABLE_INFO(
 static GList*
 _gtk_cell_layout_get_cells(GtkTreeViewColumn *column)
 {
-#if GTK_CHECK_VERSION(2, 12, 0)
```

Comment:
Why not remove this whole function? It looks like it was only added for backwards compatibility pre-2.12, and has a confusing name.

Reply:
Done in #725.